src/components/organisms/AssetActions/Pool/Remove.tsx

```diff
@@ -149,13 +156,41 @@ export default function Remove({
     totalPoolTokens
   ])
 
+  async function computeOceansRemoved(amountPoolShares: string) {
```

Comment:
small naming thing but `compute` is kinda reserved in our tech stack for something different. We could use `calculate` instead